docs/installation.rst

```diff
@@ -23,7 +23,7 @@ Using the inline require for composer:
 
 .. code-block:: sh
 
-	composer require friendsofcake/crud:~4.0
+	composer require friendsofcake/crud:~4.2
```

Comment:
This can actually just be `composer require friendsofcake/crud` composer will automatically infer the correct latest version.